fv3core/stencils/a2b_ord4.py

```diff
@@ -11,11 +11,12 @@
     sqrt,
 )
 
+import fv3core._config as spec
```

Comment:
When we talked I thought you had wanted to pass the legacy grid object as an argument to remove this? I don't object-- I see here they are used for getting the grid variables we actually want to compute in this module. So it could be a handy way to search and find those. Just want to check I am understanding the intention correctly here?

Reply:
Yes, that was the original plan. This will make it easier to search for those, and also let me rely on "grid is not an argument to the object" as a marker that the object has at least undergone the initial round of init refactors.